### PR TITLE
Content updates

### LINKED ISSUE #2288: [BUG] App sub tabs and titles not sentence case
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Right now, sub tabs and titles of SR sections are title case (each word is capitalized) but they should all be sentence case (only first word capitalized), unless it's a name/proper noun. Sentence case is best practice, in our style guide, and easier for users to read. 

Screen Shot 2021-08-12 at 5 01 39 PM

### Steps To Reproduce

1. Go to app
2. Click on gear icon to view Manage users related sub tabs. Click on different tabs to view section titles (People ---> Date of Birth needs the "b" lowercased, Results needs the R in "Test Results" lowercased, etc.). Click on all tabs to double check titles below.  

### Expected Behavior

All sub tabs and section titles(Manage Users, etc.) should all be sentence case (only first word capitalized), unless it's a name/proper noun. For example: "Manage users" not "Manage Users" etc.

### Additional context

_No response_

### Does this issue have a priority label?

- [X] I have added a priority label

### Has issue been assigned to a team?

- [X] I have added a team label

### PR DESCRIPTION
**Pull request by nullflux**
## Related Issue or Background Info

- Resolves #2288
- Paired with @Rebecca-LM to fix a number of content issues

### UI
- [x] Any changes to the UI/UX are approved by design 
- [x] Any new or updated content (e.g. error messages) are approved by design 

### Testing
- [x] Includes a summary of what a code reviewer should verify

### Changes are Backwards Compatible
- [x] Database changes are submitted as a separate PR
  - [x] Any new tables that do not contain PII are accompanied by a GRANT SELECT to the no-PHI user
  - [x] Any changes to tables that have custom no-PHI views are accompanied by changes to those views
        (including re-granting permission to the no-PHI user if need be)
  - [x] Liquibase rollback has been tested locally using `./gradlew liquibaseRollbackSQL` or `liquibaseRollback`
- [x] GraphQL schema changes are backward compatible with older version of the front-end

### Security
- [x] Changes with security implications have been approved by a security engineer (changes to  authentication, encryption, handling of PII, etc.)
- [x] Any dependencies introduced have been vetted and discussed

## Cloud
- [x] DevOps team has been notified if PR requires ops support
- [x] If there are changes that cannot be tested locally, this has been deployed to our Azure `test` environment for verification

### CODE REVIEW

**Review by Rebecca-LM — CHANGES_REQUESTED** [review]
Looks good! Can we change "This is a training site with sample data for training purposes only." to "for demonstration purposes only"? All else is good to go on my end, as far as I can tell. @nullflux

**Comment by nullflux** [comment]
Thanks for the review @Rebecca-LM! Updated in 5ef5b449

**Review by Rebecca-LM — APPROVED** [review]
LGTM and I'll take another look after it goes live!

File: frontend/src/app/Settings/ManageOrganization.tsx [hunk 1]
```diff
@@ -7,37 +7,10 @@ import RequiredMessage from "../commonComponents/RequiredMessage";
...
-  { value: "urgent_care", label: "Urgent Care" },
-  { value: "nursing_home", label: "Nursing Home" },
-  { value: "treatment_center", label: "Substance Abuse Treatment Center" },
-  { value: "hospice", label: "Hospice" },
-  { value: "pharmacy", label: "Pharmacy" },
-  { value: "lab", label: "Lab" },
-  { value: "other", label: "Other" },
-];
-
-interface Props {
```
Comment (nickscialli-usds) [inline frontend/src/app/Settings/ManageOrganization.tsx:35]: Maybe deleted too much here? The app won't build for me because there  is no `Props` type

**Review by nickscialli-usds — APPROVED** [review]
LGTM after adding props back in for the one component

File: frontend/src/app/Settings/ManageOrganization.tsx (hunk 1 above)
Comment (nullflux) [reply frontend/src/app/Settings/ManageOrganization.tsx:35]: Oh jeez, thank you!

**Pull request merged**